src/Containers/MiddleCanvas/DualSelectRadio.tsx

```diff
@@ -147,15 +147,15 @@ interface SelectContainerProps {
 };
 const SelectContainer = styled.div<SelectContainerProps>`
     ${scroll};
-    ${({ theme, isVisible }): string => `
-        border: ${isVisible ? `solid 1px ${theme.colors.input.default}` : ''};
-        background-color: ${isVisible ? theme.colors.background : ''};
-        ${isVisible ? media(
+    width: 29%;
+    ${media(
         'tablet',
         `
-            width: 95%;
-            height: 220px;
-        `) : '180px'}; 
+            width: 95%
+        `)}; 
```

Comment:
A bit weird you didn't just say
width 95%
then only on Phone make the width smaller?

Becuase logically,
Tablet, Desktop Everthing Else is width 95% and only on phone is width 29%.